test/pco_api/people/report_test.exs

```diff
@@ -0,0 +1,92 @@
+defmodule PcoApi.People.ReportTest do
+  use ExUnit.Case, async: false
+  doctest PcoApi.People.Report
+  alias PcoApi.People.Report
+  alias TestHelper.Fixture
+
+  setup do
+    bypass = Bypass.open
+    Application.put_env(:pco_api, :endpoint_base, "http://localhost:#{bypass.port}/")
+    {:ok, bypass: bypass}
+  end
+
+  test ".list requests the v2 endpoint", %{bypass: bypass} do
+    Bypass.expect bypass, fn conn ->
+      assert "/people/v2/reports" == conn.request_path
+      assert "GET" == conn.method
+      Plug.Conn.resp(conn, 200, Fixture.read("report.json"))
+    end
+    Report.list
+  end
+
+  test ".list returns a list of Record structs", %{bypass: bypass} do
+    Bypass.expect bypass, fn conn ->
+      Plug.Conn.resp(conn, 200, Fixture.read("reports.json"))
+    end
+    assert [%PcoApi.Record{} | _rest] = Report.list
+  end
+
+  test ".get(id) returns a single record", %{bypass: bypass} do
+    Bypass.expect bypass, fn conn ->
+      assert "/people/v2/reports/1" == conn.request_path
+      Plug.Conn.resp conn, 200, Fixture.read("report.json")
+    end
+    report = Report.get(1)
+    assert %PcoApi.Record{id: "1", type: "Report"} = report
+  end
+
+  test ".created_by gets the list's creator", %{bypass: bypass} do
+    Bypass.expect bypass, fn conn ->
+      assert "/people/v2/people/1" == conn.request_path
+      Plug.Conn.resp conn, 200, Fixture.read("report.json")
+    end
+    assert %PcoApi.Record{} = bypass |> record_with_links |> Report.created_by
+  end
+
+  test ".updated_by gets list updated_by", %{bypass: bypass} do
+    Bypass.expect bypass, fn conn ->
+      assert "/people/v2/people/1" == conn.request_path
```

Comment:
Can you make this a different ID from `created_by` just to ensure the functions are getting their info from different links?